include/vcpkg/base/messages.h

```diff
@@ -2095,5 +2095,221 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Failed to parse {path}, expected a top-level object.");
-
+    DECLARE_MESSAGE(HelpTxtOptDryRun, (), "", "Do not actually build or install.");
```

Comment:
nit but, is there a reason why some of these messages end with a period and others do not? should we be consistent?

Reply:
I've been trying to add periods to actual sentences, but sometimes the messages are sentence fragments, and adding a period feels weird.